Firestore/Source/Remote/FSTDatastore.mm

```diff
@@ -301,24 +306,24 @@ - (void)invokeRPCWithFactory:(GRPCProtoCall * (^)(void))rpcFactory
                 errorHandler:(FSTVoidErrorBlock)errorHandler {
   // TODO(mikelehen): We should force a refresh if the previous RPC failed due to an expired token,
   // but I'm not sure how to detect that right now. http://b/32762461
-  [self.credentials
-      getTokenForcingRefresh:NO
-                  completion:^(Token result, NSError *_Nullable error) {
-                    error = [FSTDatastore firestoreErrorForError:error];
-                    [self.workerDispatchQueue dispatchAsyncAllowingSameQueue:^{
-                      if (error) {
-                        errorHandler(error);
-                      } else {
-                        GRPCProtoCall *rpc = rpcFactory();
-                        [FSTDatastore
-                            prepareHeadersForRPC:rpc
-                                      databaseID:&self.databaseInfo->database_id()
-                                           token:(result.is_valid() ? result.token()
-                                                                    : absl::string_view())];
-                        [rpc start];
-                      }
-                    }];
-                  }];
+  _credentials->GetToken(
+      false,  // force_refresh
```

Comment:
nit: rather than a documented boolean, consider a constant (to eliminate the magic 'false').
i.e. `creds->GetToken(kDontForceRefresh, ...)`

Reply:
The magic boolean is consistent with the other platforms, though there it's `/*force_refresh=*/false`.

Reply:
I'd argue it should be fixed everywhere then. (Though I admit, this is pretty low on the priority list.)

Since this is "no worse" then the other platforms, feel free to consider my suggestion optional.